packages/jellyfish-api-core/src/category/token.ts

```diff
@@ -81,6 +81,19 @@ export class Token {
   async getToken (symbolKey: string): Promise<TokenResult> {
     return await this.client.call('gettoken', [symbolKey], 'number')
   }
+
+  /**
+   * Get detailed information about a DeFiChain custom transaction.
+   * Will search wallet transactions and mempool transaction if a blockhash is provided
+   * and that block is available then details for that transaction can be returned.
+   * -txindex can be enabled to return details for any transaction.
+   *
+   * @param {string} txid
+   * @return {Promise<CustomTxInfo>}
+   */
+  async getCustomTx (txid: string): Promise<CustomTxInfo> {
```

Comment:
missing 2nd param
https://github.com/DeFiCh/ain/blob/df796cb7d6dbbe0ed7d051e79db0f36630f77afb/src/masternodes/rpc_tokens.cpp#L558

Reply:
<img width="887" alt="Screenshot 2021-05-17 at 11 38 51 AM" src="https://user-images.githubusercontent.com/7101806/118429578-94416d00-b704-11eb-9cb5-96ef7c59c73c.png">

No matter what I pass in as 2nd param, the test pass. Hence I was thinking maybe the comments were wrongly written. Let me investigate further again.

Reply:
yes @jingyi2811
that'd be something the test that you expected to be failed but passed
or something you found weird.. please report to channel